forgevtt-module.js

```diff
@@ -2081,20 +2131,22 @@ class ForgeVTT_FilePicker extends FilePicker {
                     <button type="button" name="forgevtt-new-folder" style="line-height: 1rem;">
                         <i class="fas fa-folder-plus"></i>New Folder
                     </button>
-                </div>`)
+                </div>`);
                 upload.hide();
-                upload.after(uploadDiv)
+                upload.after(uploadDiv);
                 uploadDiv.append(upload);
-                uploadDiv.find('button[name="forgevtt-upload"]').click((_ev) => upload.click());
-                uploadDiv.find('button[name="forgevtt-new-folder"]').click((_ev) => this._onNewFolder());
+                uploadDiv.find('button[name="forgevtt-upload"]').on("click", (_ev) => upload.click());
+                uploadDiv.find('button[name="forgevtt-new-folder"]').on("click", (_ev) => this._onNewFolder());
             }
         }
 
+        const select = html.find('select[name="bucket"]');
+        select.parent().on("change", this._onChangeBucketParent.bind(this));
```

Comment:
Here the handler - supposed to bypass the actual select change event - is attached to the parent.

Reply:
Wouldn't this callback always get called after Foundry's original `_onChangeBucket` attached to the `select` itself, since the event starts at the `select`?

Reply:
Indeed. This was the latest in a series of tries, it'll be cleaned up.
I wish to properly override both <v12 `_onBucketChange` and >v12 `#onBucketChange`, here trying to "follow up on it" instead but it isn't working as I wanted it to.